README.md

```diff
@@ -1,2 +1,80 @@
-# module-ballerina-math.vector
-Ballerina Math Module
+Ballerina Math Vector Library
+==============================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-math.vector/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerina-math.vector/actions/workflows/build-timestamped-master.yml)
+  [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-math.vector/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-math.vector)
+  [![Trivy](https://github.com/ballerina-platform/module-ballerina-math.vector/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerina-math.vector/actions/workflows/trivy-scan.yml)
+  [![GraalVM Check](https://github.com/ballerina-platform/module-ballerina-math.vector/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerina-math.vector/actions/workflows/build-with-bal-test-native.yml)  
+  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-math.vector.svg)](https://github.com/ballerina-platform/module-ballerina-math.vector/commits/main)
+  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/math.vector.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fmath.vector)
+
+This module provides functions for doing vector operations including calculating l1 and l2 norm, dot product, cosine similarity, euclidean distance and manhattan distance.
```

Comment:
```suggestion
This module provides functions for doing vector operations including calculating the `l1` and `l2` norm, dot product, cosine similarity, euclidean distance, and Manhattan distance.
```

Reply:
Addressed in https://github.com/ballerina-platform/module-ballerina-math.vector/pull/1/commits/1fe0f828ef1ea29a4fc27924af5f541af69729ff.